docs/installation.md

```diff
@@ -39,7 +39,7 @@ $ proto install bun
 
 ### Windows
 
-Bun provides a _limited, experimental_ native build for Windows. At the moment, only the Bun runtime is supported.
+Bun binaries for windows are not yet available, but coming soon. Join the [#windows](https://discord.com/channels/876711213126520882/1149339379446325248) discord channel in discord to stay up to date.
 
 - `bun <file>`
```

Comment:
I think the remainder of this section is still not relevant. The natural question is "okay so binaries for windows are not yet available. But according to this section I can run `bun <file>` - how do I do so?" I think the least confusing thing is to delete lines 44-52.